AppDashboard/static/plugins/forms/tiny_mce/plugins/spellchecker/editor_plugin_src.js

```diff
@@ -167,7 +167,7 @@
 		},
 
 		_getSeparators : function() {
-			var re = '', i, str = this.editor.getParam('spellchecker_word_separator_chars', '\\s!"#$%&()*+,-./:;<=>?@[\]^_{|}���������������\u201d\u201c');
+			var re = '', i, str = this.editor.getParam('spellchecker_word_separator_chars', '\\s!"#$%&()*+,-./:;<=>?@[\]^_{|}����������������\u201d\u201c');

```

Comment:
Not sure what this change is doing. Did your editor auto-replace these characters?